urls.py

```diff
@@ -92,6 +90,11 @@
     url(r'^submit_feedback/$', submit_feedback, name='submit_feedback'),
 ]
 
+for url_modules in extensions.get_contributions("urls:domain_specific"):
```

Comment:
Are there tests for this? I would have expected it to raise an error.
```suggestion
for url_modules in extensions.get_contributions("domain_specific_urls"):
```